adapters/http-base/src/test/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapterTest.java

```diff
@@ -923,7 +1086,7 @@ private HttpContext newHttpContext(
             final Buffer payload,
             final String contentType,
             final HttpServerRequest request,
-            final HttpServerResponse response) {
+            final HttpServerResponse response, final CommandContext commandContext) {
```

Comment:
instead of having to add a `null` parameter to all existing constructor invocations, can we overload the method instead, keeping the variant without the CommandContext parameter?